packages/curvenote/src/sync/init.ts

```diff
@@ -115,48 +118,66 @@ export async function init(session: ISession, opts: Options) {
   let pullComplete = false;
   let title = projectConfig?.title || siteConfig.title || undefined;
   if (content === 'folder') {
-    if (projectConfigPaths.length) {
+    if (projectConfigPaths.length > 0) {
       const pathListString = projectConfigPaths
         .map((p) => `  - ${join(p, CURVENOTE_YML)}`)
```

Comment:
At some point we could pull some updated functionality from myst, for example, like to get the correct existing yaml file name here, rather than just printing `curvenote.yml` always.